frontend/src/metabase/query_builder/components/filters/modals/BulkFilterSelect/BulkFilterSelect.tsx

```diff
@@ -93,7 +93,6 @@ export const BulkFilterSelect = ({
           onChangeFilter={handleChange}
           onClose={closePopover}
           checkedColor="brand"
-          commitOnBlur
```

Comment:
removed this functionality because it's not actually used by this component anymore